components/desktop/freerdp/freerdp.p5m

```diff
@@ -25,7 +25,7 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 # special permissions for xfreerdp
 file files/exec_attr path=etc/security/exec_attr.d/freerdp
 
-file path=usr/bin/freerdp-proxy
+file path=usr/bin/freerdp-proxy pkg.depend.runpath=usr/lib/$(MACH64)/freerdp2:$PKGDEPEND_RUNPATH
```

Comment:
What is the real RPATH, RUNPATH of the file? Usually you don't need this trick, if they are set correctly.

Reply:
Yeah found out CMAKE for whatever reason doesn't add /usr/lib/amd64/freerdp2 the rpath is only /usr/gcc/7/lib/amd64
so I probably try adding something similar as used in libreoffice.